packages/plugin-table/src/TableRowDomParser.ts

```diff
@@ -0,0 +1,74 @@
+import { AbstractParser } from '../../plugin-parser/src/AbstractParser';
+import { DomParsingEngine } from '../../plugin-dom/src/DomParsingEngine';
+import { TableRowNode } from './TableRowNode';
+
+export class TableRowDomParser extends AbstractParser<Node> {
+    static id = 'dom';
+    engine: DomParsingEngine;
+
+    predicate = (item: Node): boolean => {
+        return (
+            item instanceof Element &&
+            (item.nodeName === 'THEAD' || item.nodeName === 'TBODY' || item.nodeName === 'TR')
+        );
```

Comment:
Now that we were able to check that together, the predicate return type could be `item is Element` and remove `item instanceof Element` in the function so that the double parenthesis can be removed.